package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "subscribe-ui-event",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
Just realized that you bump 2.0.6 already lol, @redonkulus could you help to release 2.0.7 💙